src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -23,147 +28,120 @@ class RefOptions
     private $id;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="string", nullable=false)
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $originalId;
+    private $guid;
 
     /**
-     * @var CoreEntity
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="CoreEntity")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $optionsForCoreEntity;
+    private $projectId;
 
     /**
-     * @var Network
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="Network")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", nullable=true)
      */
-    private $optionsForNetwork;
+    private $entity;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="json_array", nullable=true)
+     * @ORM\Column(type="string", nullable=false)
      */
-    private $adminOptions;
+    private $type;
 
     /**
      * @var array
      *
      * @ORM\Column(type="json_array", nullable=true)
      */
-    private $localOptions;
-
-    /**
-     * @var string
-     *
-     * @ORM\Column(type="string", length=30, nullable=true)
-     */
-    private $projectSpace;
-
-    /**
-     * RefOptions constructor.
-     * @param string $originalId
-     * @param CoreEntity|Network $optionsFor
-     */
-    public function __construct(string $originalId, $optionsFor)
-    {
-        $this->setOriginalId($originalId);
-        $this->setOptionsFor($optionsFor);
+    private $options;
+
+    public function __construct(
+        string $entity,
+        string $guid,
+        string $projectId,
+        string $type,
+        DateTime $createdAt,
```

Comment:
You don't need to pass in createdAt and updatedAt, those properties are handled by the TimestampableEntity trait when you submit to the database.